modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/rel/AbstractNode.java

```diff
@@ -161,7 +161,7 @@ protected void checkState() throws Exception {
             throw new ExecutionCancelledException();
         }
         if (Thread.interrupted()) {
-            throw new IgniteInternalCheckedException(OPERATION_INTERRUPTED_ERR, "Thread was interrupted.");
+            throw new IgniteInternalCheckedException(RUNTIME_EXECUTION_ERR, "Thread was interrupted.");
```

Comment:
Execution nodes are run in SQL query pool.
If thread is interrupted that means pool has been stopped and query was cancelled.

So, we either never get here because of `context.isCancelled()` returns `true`, or we should throw ExecutionCancelledException, isn't it?

Reply:
Replaced with `ExecutionCancelledException `.